core/service/logging/log.py

```diff
@@ -68,6 +68,31 @@ def ensure_str(s: Any) -> Any:
         )
         if record.exc_info:
             data["traceback"] = self.formatException(record.exc_info)
+        if record.process:
+            data["process"] = record.process
+
+        # If we are running in a Flask context, we include the request data in the log
+        try:
+            from flask import request
+
+            data["request"] = {
+                "path": request.path,
+                "method": request.method,
+                "host": request.host_url,
+            }
+            if request.query_string:
+                data["request"]["query"] = request.query_string.decode()
+        except (RuntimeError, ImportError):
+            pass
+
+        # If we are running in uwsgi context, we include the worker id in the log
+        try:
+            import uwsgi
```

Comment:
Could we set up the `uwsgi` and Flask `request` objects once per instantiation on the `__init__` (or maybe even just once total on the class).

Reply:
I can give it a try... I think that will work for `uwsgi`, but I'm not sure if it will work for `flask`

Reply:
imports updated 👍🏻